<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYZImportDialog</class>
 <widget class="QDialog" name="XYZImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XYZ Import</string>
  </property>
  <widget class="QLabel" name="Filename">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonImport">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Filter</string>
   </property>
   <widget class="QLineEdit" name="endY_LineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="startX_LineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EndX</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>StartX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>StartY</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="restrict_radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Restrict from</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EndY</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endX_LineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>60</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="startY_LineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionBrowse">
   <property name="text">
    <string>Browse</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>startX_LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>startY_LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>endX_LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>endY_LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restrict_radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
